code/modules/virus2/effect.dm

```diff
@@ -13,8 +13,8 @@
 		// How damaging the virus is. Higher values are worse.
 
 	var/chance = 3
-		// Under normal conditions, the percentage chance per tick to activate.
-	var/max_chance = 6
+		// Under normal conditions, the percentage chance per tick to activate. 
+	var/max_chance = 6	
```

Comment:
???

Current one is still 50, why's this changed to 6 and why is it not showing in the files changed anymore?